Makefile

```diff
@@ -72,17 +72,20 @@ docs: ## generate Sphinx HTML documentation, including API docs
 CONTENT_URL := http://research.cs.aalto.fi/pml/software/elfi/docs/0.5/
```

Comment:
Actually there is no pics in 0.5 docs, but anyway these are going to be for the 0.6 version :)